src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -59,6 +56,20 @@ class ImportResourceTemplate extends Component {
   }
 
   createResource = async (content, group) => {
+    try {
+      // first, make sure the client instance has a valid JWT id token set
+      await CognitoUtils.getIdTokenString(this.props.authenticationState.currentUser)
+              .then((idToken) => this.instance.apiClient.authentications['CognitoUser'].accessToken = idToken)
+    } catch(error) {
+      // TODO: add auth-related error handling similar to the catch on the createResourceWithHttpInfo try below, e.g.
+      // * display a warning that the operation failed due to error with current session
+      //   * add action and reducer for session expired, have LoginPanel display specific err msg.  reducer should clear session (and
+      //     user as needed) so the login panel goes back to the not logged in state.  highlight LoginPanel or something too (along w/ the err msg).
+      //   * user logs in via login panel again.  app state now has a valid cognito user with a valid session.
+      //   * user tries to do this action again, now that they have a valid sesison, hopefully succeeds.
```

Comment:
captured in #528, though this will almost definitely be broken off into its own ticket